medusa/server/core.py

```diff
@@ -150,6 +151,8 @@ def __init__(self, options=None, io_loop=None):
             (r'{base}/login(/?)'.format(base=self.options['web_root']), LoginHandler),
             (r'{base}/logout(/?)'.format(base=self.options['web_root']), LogoutHandler),
 
+            (r'{base}/token(/?)'.format(base=self.options['web_root']), TokenHandler),
```

Comment:
This was already answered as being fine in slack. The only thing I'm awaiting changes on is the ietf language codes for filenames.